Cargo.toml

```diff
@@ -44,30 +49,27 @@ codec = { package = "parity-scale-codec", version = "1.3", default-features = fa
 hash-db = "0.15"
 
 # Substrate
-sp-database = { git = "https://github.com/paritytech/substrate", branch = "master", package="sp-database" }
-sp-storage = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-storage" }
-sp-externalities = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-externalities" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-runtime" }
-sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-client-api" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-blockchain" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-core" }
-sp-api = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-api" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-block-builder" }
-sp-version = {  git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-version" }
-sc-executor = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-executor" }
-sc-chain-spec = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-chain-spec" }
-sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-trie" }
-sp-state-machine = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-state-machine" }
-sp-io = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-io" }
-itoa = "0.4.6"
-include_dir = "0.6.0"
-tempfile = "3.1.0"
-async-executor = "0.1.2"
-arc-swap = "0.4.7"
+sp-database = { git = "https://github.com/paritytech/substrate", package="sp-database" , rev = "35fe3cd1bc4b64cadb0bc6196ae40173db65bb28" }
```

Comment:
rev `35fe3cd1bc4b64cadb0bc6196ae40173db65bb28` is rather old (Aug 26) – what is stopping us from using `master` or `2.0`, i.e. what motivated the change?